addOns/postman/src/main/javahelp/org/zaproxy/addon/postman/resources/help/map.jhm

```diff
@@ -0,0 +1,9 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!DOCTYPE map
+  PUBLIC "-//Sun Microsystems Inc.//DTD JavaHelp Map Version 1.0//EN"
+         "http://java.sun.com/products/javahelp/map_1_0.dtd">
+
+<map version="1.0">
+	<mapID target="postman-icon" url="contents/images/postman.png" />
+    <mapID target="postman" url="contents/postman.html" />
```

Comment:
Use the prefix `addon.*` for the targets.

Reply:
The other `target` references need to be updated to match the new value.

Reply:
@thc202 Done